src/main/resources/default/taglib/k/link.html.pasta

```diff
@@ -1,11 +1,13 @@
+<i:arg type="String" name="id" default="" description="Defines the id."/>
 <i:arg type="String" name="link" description="Defines a url or link."/>
 <i:arg type="String" name="name" default="" description="Defines a speaking name or description."/>
 <i:arg type="String" name="icon" default="fas fa-external-link-alt" description="Defines the icon left to the link name."/>
+<i:arg type="String" name="target" default="_blank" description="Defines the target where to display the linked url or link."/>
```

Comment:
`_self` seems to be a more obvious default ... or empty, and then `isFilled`

Reply:
Would also prefer a non breaking version of this to not have to check all places where the link tag is used.

Reply:
okay then I am going to put "_self" as default behavior

Reply:
done